CIP-NFTMetadataStandard.md

```diff
@@ -0,0 +1,119 @@
+---
+CIP:
+Title: NFT Metadata Standard
+Authors: Alessandro Konrad <alessandro.konrad@live.de>, Smaug <smaug@pool.pm>
+Comments-URI:
+Status: Draft
+Type: Informational
+Created: 2021-04-08
+Post-History: https://forum.cardano.org/t/cip-nft-metadata-standard/45687 and https://www.reddit.com/r/CardanoDevelopers/comments/mkhlv8/nft_metadata_standard/
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This proposal defines an NFT Metadata Standard for Native Tokens.
+
+## Motivation
+
+Tokens on Cardano are a part of the ledger. Unlike on Ethereum, where metadata can be attached to a token through a smart contract, this isn't possible on Cardano because tokens are native and Cardano uses a UTxO ledger, which makes it hard to directly attach metadata to a token.
+So the link to the metadata needs to be established differently.
+Cardano has the ability to send metadata in a transaction, that's the way we can create a link between a token and the metadata. To make the link unique, the metadata should be appended to the same transaction, where the token forge happens:
+
+> Given a token in a EUTXOma ledger, we can ask “where did this token come from?” Since tokens
+> are always created in specific forging operations, we can always trace them back through their
+> transaction graph to their origin.
+
+(Section 4.1 in the paper: https://hydra.iohk.io/build/5400786/download/1/eutxoma.pdf)
+
+## Considerations
+
+That being said, we have unique metadata link to a token and can always prove that with 100% certainty. No one else can manipulate the link except if the policy allows it to ([update mechanism](#update-metadata-link-for-a-specific-token)).
+
+## Specification
+
+This is the registered `transaction_metadatum_label` value
+
+| transaction_metadatum_label | description  |
+| --------------------------- | ------------ |
+| 721                         | NFT Metadata |
+
+### Structure
+
+The structure allows for multiple token mints, also with different policies, in a single transaction.
+
+```
+{
+  "721": {
+    "<policy_id>": {
+      "<asset_name>": {
+        "name": "<name>",
+        "image": "<uri>",
```

Comment:
If including an image link, it could be a good idea to also include a hash of the image data inside the metadata itself so that one can verify that the linked image does indeed corresponds to what the author originally linked.